avocado/utils/lv_utils.py

```diff
@@ -343,9 +343,10 @@ def vg_create(vg_name, pv_list, force=False):
         pv_list = " ".join(pv_list)
     else:
         pv_list = str(pv_list)
-    cmd += f" {vg_name} {pv_list}"
     if force:
-        cmd += f"{cmd} -f -y"
+        cmd += f" {vg_name} {pv_list} -f -y"
+    else:
+        cmd += f" {vg_name} {pv_list}"
```

Comment:
@PavamanSubramaniyam I think it doesn't need so many changes here.
just removing of `+` in existing code of force section will do the things. can you please re-check on this

Reply:
We need to include + here for the force section since we have an earlier line where in the cmd variable is set to vgcreate command. So, we need to append the parameters to vgcreate command which is done by using the the + in the force section.

Earlier in the original code for this force section, the cmd value was getting replaced twice and the full command was failing to create the volume group using the logical volume passed.

Reply:
@PavamanSubramaniyam I am suggesting to just remove the + from original code only. and do not add your code. it is not required. just remove + from original code and re-push the PR. I have verified by removing + from original code and it is working. 

So you also please verify by removing just + from original code and re-push the code.